### ISSUE TITLE
The cps_weights.csv and puf_weights.csv files have different formats

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/taxdata#176]
The non-standard format of `cps_weights.csv` is causing substantial problems in Tax-Calculator.
What's the timeline for making the `cps_weights.csv` file have the same format as the `puf_weights.csv` file?

Here is the format difference:
```
$ gunzip -k puf_weights.csv.gz 
$ head -2 puf_weights.csv
WT2011,WT2012,WT2013,WT2014,WT2015,WT2016,WT2017,WT2018,WT2019,WT2020,WT2021,WT2022,WT2023,WT2024,WT2025,WT2026,WT2027
147096,146655,149097,148523,150406,152774,155407,157834,159967,161850,163762,165601,167454,169323,171220,173147,175074

$ gunzip -k cps_weights.csv.gz 
$ head -2 cps_weights.csv
SEQUENCE,WT2015,WT2016,WT2017,WT2018,WT2019,WT2020,WT2021,WT2022,WT2023,WT2024,WT2025,WT2026,WT2027
100,23002,23312,23627,23946,24269,24597,24929,25265,25606,25952,26302,26658,27017
```
The `cps_weights.csv` file is missing `WT2014` and has instead a mysterious `SEQUENCE` variable.

For an earlier discussion of this issue, see issue #125.

**Comment by andersonfrailey** [comment]
@martinholmer, I can get a PR open to drop the sequence variable now. As for the 2014 weights column, I can copy over the weights from the CPS file, but that seems somewhat redundant. I still haven't received a definitive answer from John about the duplicate weights issue I brought up in issue #125.

**Comment by martinholmer** [comment]
@andersonfrailey said:
> I can get a PR open to drop the sequence variable now. As for the 2014 weights column, I can copy over the weights from the CPS file, but that seems somewhat redundant. I still haven't received a definitive answer from John about the duplicate weights issue I brought up in issue #125.

I don't believe the `s006` values in the `cps.csv.gz` file.  So, don't do anything now.

Instead, focus on figuring out what the 2014 weights **should** be and then update the weights file.

**Comment by andersonfrailey** [comment]
@martinholmer got it. I'll go ahead and remove the sequence variable and keep working on the 2014 weights issue.

**Comment by martinholmer** [comment]
@andersonfrailey said in issue #176 conversation:
> As for the [missing] 2014 weights column [in `cps_weights.csv.gz` file], I can copy over the weights from the CPS file [multiplying by 100 and rounding them to the nearest integer].

Actually, on second thought, this would be a big help to Tax-Calculator.
You can do this even though #145 is still unresolved, right?

**Comment by andersonfrailey** [comment]
@martinholmer asked:
>Actually, on second thought, this would be a big help to Tax-Calculator.
You can do this even though #145 is still unresolved, right?

Yes. I can open up a PR that does this today.

**Comment by martinholmer** [comment]
Issue #176 has been resolved by pull request #184.
